web/themes/custom/fluffiness/css/layout/hospital.css

```diff
@@ -0,0 +1,145 @@
+@import "layout/breakpoints.css";
+
+.node__title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+}
+
+.node__content {
+  margin-top: 32px;
+}
+
+.hospital__date, .hospital__image-alt {
+  color: var(--dark-grey);
+}
+
+/*blockquote*/
+blockquote {
```

Comment:
be careful when using such generic selectors such as this one, or `p` `a` `h2`etc as, when this file loads on a page, it will apply these styles to any .blockquote present